### DISCUSSION TITLE
why windows defender think there is trojan in arm64 107.46

### DISCUSSION THREAD

**Discussion by YBS-PC** [AdguardTeam/AdGuardHome#6848] [General]
why windows defender think there is trojan in arm64 107.46
Trojan:Script/Wacatac.H!ml
AdGuardHome_linux_arm64\AdGuardHome\AdGuardHome

**Comment by ghost** [comment]
It's a false positive.
You can report it [here](https://www.microsoft.com/en-us/wdsi/filesubmission).
